setup.py

```diff
@@ -137,13 +138,14 @@ def find_version():
     # Optional
     install_requires=[
         'pytz',
+        'pydatum==0.2.3',
```

Comment:
can this be replaced by an existing python package ?

Reply:
Not without lots of refactoring at the moment. We can raise an issue to do that later but I would prefer to keep this one for now.

Reply:
can you just add the pydatum file in the project then and remove it later ?

Reply:
I extracted it into a separate package because it is used in several projects. Do you have specific concerns in this regard?

Reply:
No CI, no doc, other datetime packages (with docs, tests, and a community) would be a better dependency.
As it is only used in the caching logic and the caching logic may be better implemented with existing caching libraries in Python, not really find of adding such dependency

Reply:
I started pydatum specifically for this purpose, so it is best suited for the purpose. It is already wrapping some useful date libraries and provides a higher-level unified and convenient methods, similar to Calendar library in Java.